docs/section11/11-3.md

```diff
@@ -0,0 +1,63 @@
+# Service_Water_Heating - Rule 11-3
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-3
+
+**Rule Description:** "Where no service water-heating system exists or has been designed and submitted with design documents but the building will have service water-heating loads, a service water-heating system shall be modeled that matches the system type in the baseline building design"
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, c
+
+**Evaluation Context:** P-RMD each SHW bat  
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements
+- **get_SHW_types_and_spaces**
```

Comment:
I think this funciton is now called get_spaces_associated_with_each_SWH_bat

Reply:
removed reference as the function is no longer used.  Updated all function references so they're current.